datacatalog/cloud-client/src/main/java/com/example/datacatalog/CreateFilesetEntryQuickStart.java

```diff
@@ -0,0 +1,171 @@
+/*
```

Comment:
Since this is a "sample application" (used in tutorial, not snippets), please place it in a separate directory with a separate pom.xml file. e.g. `data-catalog/quickstart`

Reply:
sure